src/test/java/bio/terra/workspace/service/status/WorkspaceManagerStatusServiceTest.java

```diff
@@ -54,4 +57,17 @@ public void testCriticalFailureNotOk() throws Exception {
         statusService.getCurrentStatus().getSystems();
     assertFalse(subsystemStatus.get("Sam").isOk());
   }
+
+  @Test
+  public void testBufferCriticalFailureNotOk() throws Exception {
```

Comment:
you probably want a test case for the happy path too.

Reply:
That is above - it is everyone's happy path :)